codelabs/codelab-02/README.md

```diff
@@ -427,15 +431,15 @@ Note that if you kill the testing program, it won't clean up after itself. If yo
 $ aws s3 ls | grep "test-bucket" | cut -f3 -d ' ' | xargs -I {} aws s3 rb 's3://{}'
 ```
 
-Feel free to install any packages you would like to use. Just make sure to add them to the `requirements.txt` file (run `pip freeze > requirements.txt`).
+_Don't stop now, you're almost there, read on_
 
 #### Host a Static Website
 
 As mentioned in class, you can use S3 to host static content, like static websites. Let's do that for the STICs website!
 
 Now that you've finished the assignment, you have a tool that you could use to upload static content for a website, however the tool does not set the Content-Type of each file when they are uploaded (they default to a byte stream type). If you tried to open an HTML file in your browser that was hosted on S3 with this Content-Type, then your browser would just download the file instead of serving it (it doesn't use the file ending to determine the type of file, as you might expect, it instead uses the Content-Type header in the response).
 
-Therefore you have two options: 1) update your tool to detect and set the Content-Type header (use a library like [`python-magic`](https://github.com/ahupp/python-magic)). Otherwise, you can use the `aws s3 sync` command.
+Here's how you check the content type. Look it's octet-stream! It's just 2-3 lines to use a library like [`python-magic`](https://github.com/ahupp/python-magic) to detect and set the content type. Update your tool to detect and set the Content-Type header.
```

Comment:
I wouldn't use that word for word, just meant it as an overview of what this section should describe. Doesn't describe how to check content-type/etc.

Reply:
(This is probably a 2-3 pgh description)